src/bundle-manifest.ts

```diff
@@ -148,3 +195,19 @@ export function mergeBundles(bundles: Bundle[]): Bundle {
   }
   return newBundle;
 }
+
+/**
+ * A simple function for generating shared bundle names based on a counter.
+ */
+export function sharedBundleUrlMapper(bundles: Bundle[]): UrlString[] {
+  let counter = 0;
+  const urls: UrlString[] = [];
+  for (let bundle of bundles) {
```

Comment:
I did not realize you could const this.  I have lots of `for (let x of xs)` and `let x` inside of for loops that I can const.  CONST ALL THE THINGS!  Done.